.env

```diff
@@ -3,8 +3,10 @@ DATABASE_USER=docker
 DATABASE_PASSWORD=docker
 DATABASE_DB=airbyte
 CONFIG_ROOT=/data
+WORKSPACE_ROOT_PARENT=/tmp
```

Comment:
before you lose you mind about this go read comments in `airbyte-config/init/create_mount_directories.sh`

Reply:
can you put these two new terrible variables at the end of the file and prefix them with HACK_ with a comment that links to an issue?